src/System.Net.WebSockets.WebSocketProtocol/tests/WebSocketProtocolTests.cs

```diff
@@ -108,6 +101,38 @@ public void CreateFromStream_ValidBufferSizes_Succeed(int bufferSize)
             }
         }
 
+        [Fact]
+        public static async Task ManagedWebSocket_ReceiveUTF8SplitAcrossMultipleBuffers()
+        {
+            // 1 character - 2 bytes
+            var payload = Encoding.UTF8.GetBytes("\u00E6");
```

Comment:
Nit: `var` => `byte[]`.  In the corefx repo, `var` is only meant to be used when the type is specified clearly on the right-hand side, which generally just means `new` expressions, explicit casts, and `default(Type)` expressions.